server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BspProjectMapper.java

```diff
@@ -57,203 +57,203 @@
 
```

Comment:
what happened in this file? is it only formatting or something more?

Reply:
infer capabilities updated.